core/src/main/java/com/sequenceiq/cloudbreak/controller/v4/ImageCatalogV4Controller.java

```diff
@@ -142,4 +144,11 @@ public ImagesV4Response getImagesByName(Long workspaceId, @ResourceName String n
         Images images = imageCatalogService.getImagesByCatalogName(workspaceId, name, stackName, platform);
         return converterUtil.convert(images, ImagesV4Response.class);
     }
+
+    @Override
+    @CheckPermissionByResourceName(action = AuthorizationResourceAction.DESCRIBE_IMAGE_CATALOG)
+    public ImageV4Response getImageByNameAndId(Long workspaceId, @ResourceName String name, String imageId) throws Exception {
```

Comment:
can we rename it to getImageByCatalogNameAndImageId to make its name more specific?

Reply:
sure. It is done